core/arch/arm/plat-stm32mp2/conf.mk

```diff
@@ -0,0 +1,74 @@
+flavor_dts_file-257F_EV = stm32mp257f-ev1.dts
+
+flavorlist-MP25 = $(flavor_dts_file-257F_EV)
+
+ifneq ($(PLATFORM_FLAVOR),)
+ifeq ($(flavor_dts_file-$(PLATFORM_FLAVOR)),)
+$(error Invalid platform flavor $(PLATFORM_FLAVOR))
+endif
+CFG_EMBED_DTB_SOURCE_FILE ?= $(flavor_dts_file-$(PLATFORM_FLAVOR))
+endif
+CFG_EMBED_DTB_SOURCE_FILE ?= stm32mp257f-ev1.dts
```

Comment:
I think using `flavor_dts_file-257F_EV` is more appropriate here (in case the dts filename changes

Reply:
If the name changes, I'd still have to change the name of the flavorlist so I think it's ok